### PR TITLE
readme and code comments

### PR DESCRIPTION
**Pull request by flossypurse**
Just fixing a couple of README things and added some comments here and there.

### CODE REVIEW

File: workflows/employment_verification.go [hunk 1]
```diff
@@ -74,6 +76,10 @@ func waitForEmploymentVerificationSubmission(ctx workflow.Context) (*types.Emplo
 }
 
 // @@@SNIPSTART background-checks-employment-verification-workflow-definition
+
+// EmploymentVerification is a Workflow Definition that is executes a Side Effect, and an  Activity,
```
Comment (robholland) [inline workflows/employment_verification.go:80]:
```suggestion
// EmploymentVerification is a Workflow Definition that executes a Side Effect, and an Activity,
```

File: workflows/employment_verification.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ func chooseResearcher(ctx workflow.Context, input *types.EmploymentVerificationW
 	return researcher, err
 }
 
+// emailEmploymentVerificationRequest simply encapsulates the logic that executes the Activity.
```
Comment (robholland) [inline workflows/employment_verification.go:38]: Not sure about the use of the word `simply` here, I'd drop that, but up to you.

**Review by robholland — APPROVED** [review]
Looks good, just a few comments.

File: workflows/background_check.go [hunk 3]
```diff
@@ -137,9 +137,11 @@ func (w *backgroundCheckWorkflow) waitForSearches(ctx workflow.Context) {
 }
 
 // @@@SNIPSTART background-checks-main-workflow-definition
-// BackgroundCheck is the main background check workflow.
-// It accepts an email as the input.
-// All other personal information for the candidate is provided by the candidate when accepting the background check.
+
+// BackgroundCheck is a Workflow Definiton that executes a variable set of Activities and Child Workflows.
```
Comment (flossypurse) [inline workflows/background_check.go:141]:
```suggestion
// BackgroundCheck is a Workflow Definition that calls for the execution of a variable set of Activities and Child Workflows.
```

File: workflows/employment_verification.go [hunk 4]
```diff
@@ -15,6 +15,7 @@ const (
 	ResearchDeadline                       = time.Hour * 24 * 7
 )
 
+// chooseResearcher simply encapsulates the logic that randomly chooses a Researcher using a Side Effect.
```
Comment (flossypurse) [inline workflows/employment_verification.go:18]:
```suggestion
// chooseResearcher encapsulates the logic that randomly chooses a Researcher using a Side Effect.
```

File: workflows/employment_verification.go (hunk 2 above)
Comment (flossypurse) [inline workflows/employment_verification.go:38]:
```suggestion
// emailEmploymentVerificationRequest encapsulates the logic that calls for the execution an Activity.
```

File: workflows/employment_verification.go [hunk 5]
```diff
@@ -74,6 +76,10 @@ func waitForEmploymentVerificationSubmission(ctx workflow.Context) (*types.Emplo
 }
 
 // @@@SNIPSTART background-checks-employment-verification-workflow-definition
+
+// EmploymentVerification is a Workflow Definition that is executes a Side Effect, and an  Activity,
+// but then waits on a Signal. It is also capable of handling a Query to get Candidate Details.
```
Comment (flossypurse) [inline workflows/employment_verification.go:81]:
```suggestion
// but then waits on and handles a Signal. It is also capable of handling a Query to get Candidate Details.
```

File: workflows/ssn_trace.go [hunk 6]
```diff
@@ -8,6 +8,9 @@ import (
 )
 
 // @@@SNIPSTART background-checks-ssn-trace-workflow-definition
+
+// SSNTrace is a Workflow Definition that executes a single Activity.
```
Comment (flossypurse) [inline workflows/ssn_trace.go:12]:
```suggestion
// SSNTrace is a Workflow Definition that calls for the execution of a single Activity.
```

**Pull request merged**